src/libs/import.c

```diff
@@ -1702,7 +1816,8 @@ const struct
   {"session/base_directory_pattern",    "base_pattern",       DT_STRING},
   {"session/sub_directory_pattern",     "sub_pattern",        DT_STRING},
   {"session/filename_pattern",          "filename_pattern",   DT_STRING},
-  {"ui_last/import_initial_rating",     "rating",             DT_INT}
+  {"ui_last/import_initial_rating",     "rating",             DT_INT},
+  {"ui_last/import_select_new",         "select_new",         DT_BOOL}
```

Comment:
If `select new` is not in presets, it should be removed from here.

Reply:
Done.